<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="COMPUTERHOME" failures="2" tests="6" name="Surefire test" time="0.083" errors="0" timestamp="2022-03-18T13:15:17 YEKT">
  <testcase classname="PersonTest" name="testPerson" time="0.017">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at PersonTest.testPerson(PersonTest.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testPerson -->
  <testcase classname="PersonTest" name="testPerson" time="0.0"/>
  <testcase classname="PersonTest" name="testPerson" time="0.0"/>
  <testcase classname="PersonTest" name="testPerson" time="0.0"/>
  <testcase classname="PersonTest" name="testPerson" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at PersonTest.testPerson(PersonTest.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- testPerson -->
  <testcase classname="PersonTest" name="testPerson" time="0.0"/>
</testsuite> <!-- Surefire test -->
